service/src/main/java/org/cbioportal/service/impl/MolecularProfileServiceImpl.java

```diff
@@ -163,18 +163,17 @@ public BaseMeta getMetaMolecularProfilesInStudies(List<String> studyIds) {
 
         //TODO: remove mutation profile once fusions are migrated to structural variant in the database
         getMolecularProfilesInStudies(studyIds, "SUMMARY").stream().forEach(molecularProfile -> {
-            if (!mapByStudyId.containsKey(molecularProfile.getCancerStudyIdentifier())) {
-                if (molecularProfile.getMolecularAlterationType()
-                        .equals(MolecularProfile.MolecularAlterationType.MUTATION_EXTENDED)
-                        || (molecularProfile.getMolecularAlterationType()
-                                .equals(MolecularProfile.MolecularAlterationType.STRUCTURAL_VARIANT)
-                                && molecularProfile.getDatatype() == "SV")) {
+            if (molecularProfile.getMolecularAlterationType().equals(MolecularProfile.MolecularAlterationType.FUSION)
```

Comment:
We should probably document these changes with a unit test or two.

Reply:
True, but all this logic is gonna go away once the data is fixed. Not sure it its worth writing tests